README.md

```diff
@@ -1,40 +1,67 @@
 # ZenDesk App
 
-## Quick Start
+## Development Repository Branches Needed
 
-- clone this repo
-- cp `.env.sample` `.env`
-  - add your credentials to the `.env` file
-    - ZENDESK_URL - URL to zendesk account
-    - ZENDESK_USERNAME - your zendesk login username
-    - ZENDESK_API_TOKEN - generated via the following steps
-      - zendesk > Admin > CHANNELS > API > Settings
-      - Token Access > Enable
-      - Add API Token
+The current master branches have not been merged to work with this app and the following are needed until they are merged with master
 
-- `npm start` - start the node server
-- `npm run build` - builds the dist dir.  (TODO: can this be automated?  Fails if this is not run before `npm start`)
-- `npm run build:watch` (in separate shell) - watch for changing files and report typescript errors
+mm-webapp: `feature/cloud-apps`  
+mm-plugin-apps: `master`
 
-**Current Branches Needed**
+## Quick Start
 
-The current master branches have not been merged to work with this app and the following are needed until they are mergd with master
+### Zendesk and Mattermost Users (System Admin privileges required)
+
+1. Clone this repo
+1. cp `.env.sample` `.env`
+    1. `ZENDESK_URL` - set URL to your zendesk account host
+        1. Ex. `https://<subdomain>.zendesk.com`
+    1. `ZENDESK_CLIENT_SECRET` - (will be set later in the setup)
```

Comment:
I assume you want smth. like 1,2,3 instead of 1,1,1 here and below.

Reply:
Actually, the nice thing about using the `1.` throughout is the at Markdown will render the result as enumerated, based on the indentation. This is nice because you don't have to renumber any of the values when you add a new entry.  You can just add the new entry.

Reply:
If you view the rendered file [here](https://github.com/mattermost/mattermost-app-zendesk/blob/225a725863a00a8a0d2f29944ebc0d0e4f31ca3f/README.md) you can see the automatic numeration as it is rendered in the viewable readme.

Reply:
cool, didn't know that.